bc_obps/registration/fixtures/mock/operation.json

```diff
@@ -1,17 +1,15 @@
 [
   {
     "model": "registration.operation",
-    "pk": 1,
     "fields": {
       "operator": 1,
       "verified_at": "2023-10-13T15:27:00.000Z",
       "verified_by": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
       "documents": [],
-      "contacts": [],
+      "application_lead": 1,
       "name": "Operation 1",
       "type": "Single Facility Operation",
-      "naics_code": 1,
-
+      "naics_code": 19,
```

Comment:
When fixtures are created in test_models, the naics code id sequence starts at 19 (no idea why). Since there are 32 naics codes, putting 19 makes both the app and test happy